lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
Agree. What I was thinking is exposing this to the rule somehow, so that error messages can be recorded in meta area (better name), and then used inside the rule.